src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml

```diff
@@ -777,6 +778,10 @@ well as menu structures (for main menu and popup menus).
          <shortcut refid="openSourceDoc" value="Ctrl+O" disableModes="emacs"/>
          <shortcut refid="openSourceDoc" value="Meta+O" if="org.rstudio.core.client.BrowseCap.isMacintosh()"/>
 
+         <shortcut refid="openSourceDocNewColumn" value="Ctrl+X Ctrl+C" disableModes="default,vim,sublime"/>
```

Comment:
If this is meant to be an Emacs-like keybinding for the command, maybe we should use the closest emacs keybinding? I think C-x C-c means something else in Emacs, and we should use something like `C-x 5 f` for this command (but we should check with @kevinushey).

Reply:
it looks like `C-x 5 f` is taken in emacs as well. @kevinushey  do you have a recommendation?